config/locales/en.yml

```diff
@@ -1,19 +1,31 @@
 en:
-  hello: 'Hello world'
-  title: 'Title'
-  log: 'LOG'
-  goals: 'GOALS'
-  tasks: 'TASKS'
-  profile: 'PROFILE'
-  new_entry: 'Add New Log Entry'
-  create_entry: 'Create New Entry'
-  welcome: 'Welcome to journey!'
-  journey: 'Journey.'
+  alerts:
```

Comment:
could you please run `i18n-tasks health` and fix if there's any issue

Reply:
ok thanks for the suggestion didn't knew this commond, i ran it and got no errors.